packages/fork-choice/test/unit/protoArray/computeDeltas.test.ts

```diff
@@ -132,52 +132,6 @@ describe("computeDeltas", () => {
     }
   });
 
-  /**
-   * Starting Aug 2023, this test case is not valid because when an attestation is added
-   * to forkchoice, the block should come first, i.e. nextIndex should be a number
-   */
```

Comment:
maybe @tuyennhv can confirm that this is ok to be removed, I would assume it was just commented out instead of fully removed for a reason